### ISSUE TITLE
Scaling test requires at least one GridCoverage

### ISSUE THREAD

**Issue by aaime** [opengeospatial/ets-wcs20#141]
**Describe the bug**
Looking at a test run, I can see the following:

```
         Testing wcs2:get-scal-req3-kvp type Mandatory in Test Mode with defaultResult Pass (s0001/d14820e89_1/d4566e24999_1)...
            Assertion: Req3:Test GridCoverage,RectifiedGridCoverage.
            [FAILURE] The service should offer at least one GridCoverage.
            [MESSAGE] The server does not support the gml coverage.
         Test wcs2:get-scal-req3-kvp Failed
```

Now, the server in question only has RectifiedGridCoverage, it does not support general GridCoverage. 
The test is however written to fail if at least one is not found:

https://github.com/opengeospatial/ets-wcs20/blob/master/src/main/scripts/ctl/ext_get-scal.xml#L357

I don't think this is what the ATS requires... requirement 3 states:

> The coverage identifier in a GetCoverage request shall address a coverage which is a GridCoverage, RectifiedGridCoverage, ReferenceableGridCoverage, or a subtype of one of these."

ReferenceableGridCoverage is commented out through the tests, but I think in order to pass the test a server should offer at least a GridCoverage or a RectifiedGridCoverage, but not both?

**Comment by pebau** [comment]
@aaime yes, you are right: the intention of the test was that at least one coverage of whatever type, dimension, range type must be provided for executing a test. So indeed it is an "or".

BTW, GridCoverage is a deprecated initial attempt dating back 20+ years and known to be flawed. CIS 1.1 adds a GeneralGridCoverage which unifies it all, and to add this into the game WCS 2.1 has been issued (no further technical change).

**Comment by aaime** [comment]
@pebau thanks for confirming. I'll see if I can issue a pull request to fix the issue then.

**Comment by aaime** [comment]
Tentative fix here: https://github.com/opengeospatial/ets-wcs20/pull/144

Solves the issue when running the test suite from the Web UI. 
I have encountered an odd problem when running the same through the REST API, reported as #143

**Comment by pebau** [comment]
@ghobona I think we can close this issue?

**Comment by petersmythe** [comment]
Has this been deployed to PROD https://cite.opengeospatial.org/teamengine yet?  According to me, the fix was not included in v5.6.1, which is still the latest version deployed.
